ci/scripts/utils/ci_utils.sh

```diff
@@ -155,3 +155,25 @@ function publish_logs() {
     fi
     echo "${URL}"
 }
+
+function cleanup_experiment() {
+    # cleanup_experiment function
+    # This function takes a directory path of an EXPDIR and
+    # gets HOMEDIR value from config.base to
+    # clean up the archived files on disk.
+
+    local PSLOT_PATH
+    local pslot
+    local HOMEDIR
+    local ARCHIVEDIR
+
+    PSLOT_PATH="$1"
+    pslot=$(basename "${PSLOT_PATH}")
+    HOMEDIR=$(grep 'export HOMEDIR=' "${PSLOT_PATH}/config.base" | cut -d'=' -f2 | tr -d '[:space:]"' || true) || true
+    eval HOMEDIR="${HOMEDIR}"
```

Comment:
What is this `eval` doing? Isn't it already defined on the prior line? `eval` should be avoided if at all possible.

Reply:
The piping left some outer quotes around some strings so that some of the variables where not resolving without it.